pcap-haiku.c

```diff
@@ -296,6 +338,11 @@ pcapint_create_interface(const char *device, char *errorBuffer)
 		         "Interface name \"%s\" is too long.", device);
 		return NULL;
 	}
+	if (! can_be_bound(device)) {
+		snprintf(errorBuffer, PCAP_ERRBUF_SIZE,
+		         "Interface \"%s\" is not supported.", device);
```

Comment:
Perhaps "does not support capturing traffic" - it's "supported" in that you san presumably send packets to 127.0.0.1, but you can't see those packets in a capture in older versions of Haiku.